cip.go

```diff
@@ -131,6 +147,16 @@ func main() {
 	}
 	sc.ReadDigestsAndTags(mkRegistryListTagsCmd)
 
+	if *createManifestPtr {
+		if err := sc.CreateManifest(mfest, *createMfestOutputPtr); err != nil {
+			fmt.Fprintln(
+				os.Stderr,
+				fmt.Errorf("cannot create manifest: %s", err))
```

Comment:
We should exit with an error like `os.Exit(1)` here.

Reply:
Done :-)